From aa17c7c08c024e3a5b810c269e0c956a9e7d95de Mon Sep 17 00:00:00 2001
From: Fabricio Henrique Ramos <fabriciohenrique.ramos@windriver.com>
Date: Thu, 30 Sep 2021 15:13:38 -0300
Subject: [PATCH] change ClientSideError logging verbosity

Regression introduced in 16.10. Reverts the following
 upstream commit since WSME is used by SysInv-api to return ClientSideErrors,
 and in the case of CLI commands, no log history for such errors would be
 available.

Reverting commit 94cd1751c7b028898a38fda0689cfce15e2a96e2

  Author: Chris Dent <chdent@redhat.com>
  Date: Thu Apr 9 14:04:32 2015 +0100

    Change client-side error logging to debug

    A client-side error (that is something akin to a 4xx HTTP response
    code) is something that is common, it is not something that should
    cause WARNING level log messages. This change switches to using
    DEBUG so that it is easier to filter out the noisy messages.
---
 wsme/api.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wsme/api.py b/wsme/api.py
index b25360a..fffba02 100644
--- a/wsme/api.py
+++ b/wsme/api.py
@@ -220,7 +220,7 @@ def format_exception(excinfo, debug=False):
         faultcode = getattr(error, 'faultcode', 'Client')
         r = dict(faultcode=faultcode,
                  faultstring=faultstring)
-        log.debug("Client-side error: %s" % r['faultstring'])
+        log.warning("Client-side error: %s" % r['faultstring'])
         r['debuginfo'] = None
         return r
     else:
-- 
2.17.1

